Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3186[Validation] Avg Loss: 0.2995, Avg Dice Score: 0.8934100%[Finetuning] Epoch: 2 Avg Loss: 0.3101[Validation] Avg Loss: 0.2940, Avg Dice Score: 0.8965 100%||[Finetuning] Epoch: 3 Avg Loss: 0.3039[Validation] Avg Loss: 0.2953, Avg Dice Score: 0.8954 100% |[Finetuning] Epoch: 4 Avg Loss: 0.3003[Validation] Avg Loss: 0.2949, Avg Dice Score: 0.8928 100% ||[Finetuning] Epoch: 5 Avg Loss: 0.2981[Validation] Avg Loss: 0.2966, Avg Dice Score: 0.8907100%[Finetuning] Epoch: 6 Avg Loss: 0.2982[Validation] Avg Loss: 0.3083, Avg Dice Score: 0.8733 100%||[Finetuning] Epoch: 7 Avg Loss: 0.2945[Validation] Avg Loss: 0.3038, Avg Dice Score: 0.8784 100% |[Finetuning] Epoch: 8 Avg Loss: 0.2941[Validation] Avg Loss: 0.2968, Avg Dice Score: 0.8899 100%||[Finetuning] Epoch: 9 Avg Loss: 0.2928[Validation] Avg Loss: 0.2935, Avg Dice Score: 0.8933100%[Finetuning] Epoch: 10 Avg Loss: 0.2892143/143 [00:13<00:00, 10.31it/s]143/143 [00:14<00:00, 10.15it/s]143/143 [00:13<00:00, 10.31it/s]143/143 [00:13<00:00, 10.31it/s]143/143 [00:13<00:00, 10.22it/s]143/143 [00:13<00:00, 10.39it/s]143/143 [00:13<00:00, 10.23it/s]143/143 [00:14<00:00, 10.15it/s]143/143 [00:13<00:00, 10.24it/s]143/143 [00:14<00:00, 10.16it/s][Validation] Avg Loss: 0.2974, Avg Dice Score: 0.8901 /home/mikhelson/MedImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'TrainerTest Results - Average Loss: 0.6418, Average Dice Score: 0.7007